### PR TITLE
start of modded bosses

### PR DESCRIPTION
**Pull request by GrahamKracker**

### CODE REVIEW

File: BTD Mod Helper.sln [hunk 1]
```diff
@@ -23,14 +23,14 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{33F3CAD7-6D69-4C86-8117-F5B3BBB81BB2}.BTD6_Debug|Any CPU.ActiveCfg = BTD6_Debug|Any CPU
-		{33F3CAD7-6D69-4C86-8117-F5B3BBB81BB2}.BTD6_Debug|Any CPU.Build.0 = BTD6_Debug|Any CPU
```
Comment (Nightmare108) [inline BTD Mod Helper.sln:27]: plz fix this as well

Comment (Onixiya) [reply BTD Mod Helper.sln:27]: plz fix deez

**Comment by Nightmare108** [comment]
Sent from Yahoo Mail on Android 
 
  On Tue, Mar 21, 2023 at 2:46 PM, ***@***.***> wrote:   


@Nightmare108 what do you want me to fix?

—Can you fix the cross patrol mod for bloons td 6 because it has 89 issues and thank you for reypling.

**Comment by jrbubble** [comment]
> @Nightmare108 what do you want me to fix?



> Sent from Yahoo Mail on Android On Tue, Mar 21, 2023 at 2:46 PM, ***@***.***> wrote: @Nightmare108 what do you want me to fix? —Can you fix the cross patrol mod for bloons td 6 because it has 89 issues and thank you for reypling. Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

What cross patrol mod never heard of it

**Pull request closed without merging**